### ISSUE TITLE
PGraphics with P3D and noSmooth() does not draw anything in the first loop

### ISSUE THREAD

**Issue by amyro** [processing/processing#2245]
Hello,
i got an issue with my PGraphics buffer. If i initialize it with P3D and enable noSmooth, nothing is drawn to the buffer in the first loop. The image remains empty. When i redraw the image a second time everything is working fine. See my example code below:

```
PGraphics buf; 

public void setup() { 
 size(800, 600, P3D); 
 noLoop();
 buf = createGraphics(800, 600, P3D); 
}

public void draw() { 
  buf.beginDraw(); 
  buf.background((float)200.0); 
  buf.lights(); 
  buf.noSmooth(); 

  buf.rect(width/2, height/2, 50, 50); 

  buf.endDraw();
  buf.save("sandbox_buf.png");
 }

public void mousePressed() { 
   redraw();     
} 
```

I already opened a discussion in the forum which resulted in the conclusion that the combination of P3D and noSmooth leads to the mentioned behaviour:
http://forum.processing.org/two/discussion/1760/pgraphics-with-nosmooth-does-not-draw-anything-in-the-first-loop#Item_6

To summarize:
no P3D and noSmooth: working
P3D and smooth: working
P3D and noSmooth: not working

Thanks for your help in advance!
